pravega/pravega
# Issue 5382: Adding Java cacerts to openjdk:11.0.8-jre-slim Base Image

**Change log description**  
Adding ca-certificate to system truststore with openjdk:11.0.8-jre-slim images 

**Purpose of the change**  
Fixes #5382 

**What the code does**  
Following code changes add the ca certs on top of the openjdk:11.0.8-jre-slim image which we are using as Base Image. 

**How to verify it**  
To verify the changes exec into the ss pod and check the following
```
root@nautilus-pravega-segment-store-0:/opt/pravega# ls /etc/ssl/certs/java/cacerts
/etc/ssl/certs/java/cacerts
```
I have also deployed pravega image with these changes and have verified that reads and writes are working by running the benchmark with this pravega deployment.
